etc/kayobe/ansible/scripts/test_smartmon.py

```diff
@@ -0,0 +1,265 @@
+import unittest
+from unittest.mock import patch
+from smartmon import (
+    parse_smartctl_info,
+    parse_smartctl_attributes,
+    main,
+)
+
+class TestSmartMon(unittest.TestCase):
+    @patch('smartmon.run_command')
+    def test_parse_smartctl_info(self, mock_run_command):
+        devices_info = [
```

Comment:
It would be useful to factor this example data out into a dedicated file. Then we can have a file for each HW profile, and loop over them.